README.md

```diff
@@ -510,57 +507,57 @@ Detailed contribution instructions can be found in the [taxonomy repository](htt
 
 ### 📜 List and validate your new data
 
-You can use the `ilab taxonomy diff` command to ensure `ilab` is registering your new knowledge or skills and your contributions are properly formatted. This command displays any new or modified YAML files within your taxonomy tree. For example, the following is the expected result of a valid compositional skill contribution after adding a new skill called `foo-lang` to the freeform writing subdirectory:
+- You can use the `ilab taxonomy diff` command to ensure `ilab` is registering your new knowledge or skills and your contributions are properly formatted. This command displays any new or modified YAML files within your taxonomy tree. For example, the following is the expected result of a valid compositional skill contribution after adding a new skill called `foo-lang` to the freeform writing subdirectory:
 
-```shell
-(venv) $ ilab taxonomy diff
-compositional_skills/writing/freeform/foo-lang/qna.yaml
-Taxonomy in $HOME/.local/share/instructlab/taxonomy is valid :)
-```
+   ```shell
+   (venv) $ ilab taxonomy diff
+   compositional_skills/writing/freeform/foo-lang/qna.yaml
+   Taxonomy in $HOME/.local/share/instructlab/taxonomy is valid :)
+   ```
 
-You can also validate your entire taxonomy by performing a diff against an empty base by using the `--taxonomy-base=empty` argument:
+- You can also validate your entire taxonomy by performing a diff against an empty base by using the `--taxonomy-base=empty` argument:
 
-```shell
-(venv) $ ilab taxonomy diff --taxonomy-base=empty
-compositional_skills/general/tables/empty/qna.yaml
-compositional_skills/general/tables/editing/add_remove/qna.yaml
-...
-Taxonomy in $HOME/.local/share/instructlab/taxonomy is valid :)
-```
+   ```shell
+   (venv) $ ilab taxonomy diff --taxonomy-base=empty
+   compositional_skills/general/tables/empty/qna.yaml
+   compositional_skills/general/tables/editing/add_remove/qna.yaml
+   ...
+   Taxonomy in $HOME/.local/share/instructlab/taxonomy is valid :)
+   ```
 
 ### 🚀 Generate a synthetic dataset
 
 Before following these instructions, ensure the existing model you are adding skills or knowledge to is still running. Alternatively, `ilab data generate` can start a server for you if you provide a fully qualified model path via `--model`.
 
 1. To generate a synthetic dataset based on your newly added knowledge or skill set in [taxonomy](https://github.com/instructlab/taxonomy.git) repository, run the following command:
 
-   With GPU acceleration:
+   *With GPU acceleration (Default)*
 
    ```shell
    ilab data generate --pipeline full --gpus <NUM_OF_GPUS>
```

Comment:
the cfg has pipeline full in it, --gpus is only if they are on a gpu enabled system. I think our default should be CPU